### PR TITLE
Add a remark about the implicit implementation of Send and make a test for it 

### PR DESCRIPTION
**Pull request by tbu-**

### CODE REVIEW

File: src/libcore/sync/atomic.rs [hunk 1]
```diff
@@ -92,6 +92,7 @@ impl Default for AtomicBool {
     }
 }
 
+// Send is implicitely implemented for AtomicBool.
```
Comment (apasel422) [inline src/libcore/sync/atomic.rs]: s/implicitely/implicitly/

(and below)

Comment (tbu-) [inline src/libcore/sync/atomic.rs]: Thanks! Fixed.

**Comment by tbu-** [comment]
This was a legit failure, I fixed it.

**Comment by wthrowe** [comment]
This test already exists: [src/test/run-pass/sync-send-atomics.rs](https://github.com/rust-lang/rust/blob/8b9ada599747648cd10d9971e97ddb610712b711/src/test/run-pass/sync-send-atomics.rs)

**Comment by tbu-** [comment]
@wthrowe Ok, removed the test.

**Pull request merged**